### ISSUE TITLE
"kick start go" fails with "table installed has no column named created_at"

### ISSUE THREAD

**Issue by danielfurman** [kick-project/kick#1]
Environment:
```bash
➜  go uname -a
Linux e590 5.10.70-1-MANJARO #1 SMP PREEMPT Thu Sep 30 15:29:01 UTC 2021 x86_64 GNU/Linux

➜  go go version
go version go1.17.1 linux/amd64

➜  go kick -v
1.0.0
```

Action and result:
```bash
➜  go kick install go git@github.com:kick-project/template-go.git
➜  go kick start go ~/projects/go/kicked-go                      

2021/11/03 09:53:28 /home/daniel/projects/gopath/pkg/mod/github.com/kick-project/kick@v1.0.0/internal/resources/sync/sync.go:194 table installed has no column named created_at
[0.937ms] [rows:0] INSERT OR REPLACE INTO `installed` (`created_at`,`updated_at`,`deleted_at`,`handle`,`template`,`origin`,`url`,`vcs_ref`,`desc`,`time`) VALUES ("2021-11-03 09:53:28.521","2021-11-03 09:53:28.521",NULL,"go","","","git@github.com:kick-project/template-go.git","","Direct installation","2021-11-03 09:53:27.235")
2021/11/03 09:53:28 table installed has no column named created_at
panic: table installed has no column named created_at

goroutine 1 [running]:
github.com/kick-project/kick/internal/resources/errs.Panic({0xbe6880, 0xc00007f680})
	/home/daniel/projects/gopath/pkg/mod/github.com/kick-project/kick@v1.0.0/internal/resources/errs/errs.go:106 +0x1c5
github.com/kick-project/kick/internal/resources/sync.(*Sync).Files(0xc0002f5860)
	/home/daniel/projects/gopath/pkg/mod/github.com/kick-project/kick@v1.0.0/internal/resources/sync/sync.go:195 +0x610
github.com/kick-project/kick/internal/services/start.Start.Start({0xc000312c80, 0xc00031e1b0, 0xc0002f4f00, {0xbe5980, 0xc0002bf448}, {0xbe6240, 0xc0000b4010}, {0xbe6240, 0xc0000b4008}, {0xbee6e0, ...}, ...}, ...)
	/home/daniel/projects/gopath/pkg/mod/github.com/kick-project/kick@v1.0.0/internal/services/start/start.go:68 +0xde
github.com/kick-project/kick/internal/subcmds/startcmd.Start({0xc0000b8010, 0x3, 0x3}, 0x1)
	/home/daniel/projects/gopath/pkg/mod/github.com/kick-project/kick@v1.0.0/internal/subcmds/startcmd/startcmd.go:46 +0x187
main.main()
	/home/daniel/projects/gopath/pkg/mod/github.com/kick-project/kick@v1.0.0/cmd/kick/main.go:53 +0x1e5
➜  go 
```

Correct behaviour:
Go project should be generated with no error

**Comment by dexterp** [comment]
Hi Fenthick, thanks for trying it. I’ll attend to this bug. If you run “kick setup” in the meantime before running “kick install” that should fix it.

**Comment by danielfurman** [comment]
> If you run “kick setup” in the meantime before running “kick install” that should fix it.

```
➜  go kick setup                     
2021/11/04 10:24:01 can not migrate database: constraints not implemented on sqlite, consider using DisableForeignKeyConstraintWhenMigrating, more details https://github.com/go-gorm/gorm/wiki/GORM-V2-Release-Note-Draft#all-new-migrator
➜  go
```
No luck with that

**Comment by danielfurman** [comment]
After upgrade to 1.1.0 and running init I have following error:
```bash
➜  go kick -v
1.1.0
➜  go kick init
➜  go kick start go ~/projects/go/kicked-go
2021/11/04 10:32:58 template.go:174: Build Error: rename /tmp/kick-go-2604356363 /home/daniel/projects/go/kicked-go: invalid cross-device link
panic: Build Error: rename /tmp/kick-go-2604356363 /home/daniel/projects/go/kicked-go: invalid cross-device link

goroutine 1 [running]:
github.com/kick-project/kick/internal/utils/errutils.Epanicf({0xa22570, 0xf}, {0xc00013bd60, 0x1, 0x1})
	/home/daniel/projects/gopath/pkg/mod/github.com/kick-project/kick@v1.1.0/internal/utils/errutils/errutils.go:53 +0x6e
github.com/kick-project/kick/internal/resources/template.(*Template).Run(0xc0001464d0)
	/home/daniel/projects/gopath/pkg/mod/github.com/kick-project/kick@v1.1.0/internal/resources/template/template.go:174 +0x19c
github.com/kick-project/kick/internal/subcmds/startcmd.Start({0xc0000200d0, 0x3, 0x3}, 0xc00015b450)
	/home/daniel/projects/gopath/pkg/mod/github.com/kick-project/kick@v1.1.0/internal/subcmds/startcmd/startcmd.go:47 +0x278
main.main()
	/home/daniel/projects/gopath/pkg/mod/github.com/kick-project/kick@v1.1.0/cmd/kick/main.go:35 +0x11a
➜  go
``` 

Probably cross-filesystem rename is not possible. I think copying would resolve the issue.

**Comment by dexterp** [comment]
Hi Feni

> > If you run “kick setup” in the meantime before running “kick install” that should fix it.
> 
> ```
> ➜  go kick setup                     
> 2021/11/04 10:24:01 can not migrate database: constraints not implemented on sqlite, consider using DisableForeignKeyConstraintWhenMigrating, more details https://github.com/go-gorm/gorm/wiki/GORM-V2-Release-Note-Draft#all-new-migrator
> ➜  go
> ```
> 
> No luck with that

This error is to do with the SQLite3 database, specifically foreign key support. 
Some questions.

* If you are checking out the source code can you paste the git log for the master branch. Alternatively if you are downloading a package please point to which one you are downloading.

```
git log master | head -60
```

* What version of SQLite3 are you using?
```
# Debian based OS
sudo dpkg -l | grep sqlite
# Redhat based OS
sudo rpm -qa | grep sqlite
```

* What Linux distribution and version of the distro are you running

I will start up a VM and poke around in this version.

**Comment by dexterp** [comment]
Hi Fenthick,

Thanks for your patience. I think there will be a few bugs to iron out as for a long time it was just me using the tool. I apologise in advance I should have not released as a v1.X.X as this is more of a beta release E.G. v0.X.X

I can not reproduce your exact error but I have found another issue. When the `/tmp` folder is a different mount point, the installation of of the full folder will not complete.

* I can't reproduce `kick install ...` problem you had. It prints a clean message to run `kick setup`. I had rebased the master branch (for a long time it was just me using my tool) and have noticed some strange problems after the rebase. Try a fresh clone of the repo and see if that works.
* The cross linking error sounds like different mountpoints. What is the device that the `/tmp` directory is mounted under and the ` /home/daniel/projects/go` folder. Can you run `mount` and paste the output. 

What I have tried...
Starting from a fresh Vagrant installation.

* Ubuntu 20.04.2 LTS
* Mount points (tmp is a separate mount point)
```
mount -t ext4
/dev/mapper/vgvagrant-root on / type ext4 (rw,relatime,errors=remount-ro)
/dev/sdb on /tmp type ext4 (rw,relatime,errors=remount-ro)
```
* gcc and make installed

1. I downloaded the git repo `git clone https://github.com/kick-project/kick.git`
2. Moved the kick setup folder asside `mv ~/.kick ~/.kick.bak` to replicate a new installation
3. Ran `make depdev` to install development dependencies
4. Ran `make install` to install the go binary to ${GOPATH}/bin
5. Ran `kick install ...` prior to `kick setup` to test what happens
```
kick install go https://github.com/kick-project/template-go.git
not initialized. please run "kick setup" to initialize configuration
```
6. Ran `kick setup` to setup the ~/.kick directory
7. Ran `kick install ...` again after `kick setup`
```
kick install go https://github.com/kick-project/template-go.git
installed handle:go -> location:https://github.com/kick-project/template-go.git
```
8. Ran `kick start go myproject` under the same mount point as the /tmp filesystem
```
cd /tmp
kick start go myproject
myproject
├── cmd
│   └── myproject
│       └── myproject.go
├── goinstalls.txt
├── go.mod
├── internal
│   ├── internal.go
│   ├── options
│   │   └── options.go
│   └── version.go.in
├── Makefile
├── myproject.rb.in
├── NAME
├── nfpm.yaml.in
├── README.md
└── VERSION
```

9. ran `kick start go myrpoject` under a different mountpoint to the `/tmp` folder and found a bug.
```
cd ~
kick start go myproject
tree myproject/
myproject/
0 directories, 0 files
```

**Comment by danielfurman** [comment]
Earlier I used kick installed without checking out the repo by using `go install -v github.com/kick-project/kick/cmd/kick@v1.0.0` and `go install -v github.com/kick-project/kick/cmd/kick@latest` (which points to 1.1.0).

Today I cloned the repo and installed the tool from _master f4f6187_ and those are results (migration error):
```shell
➜  kick git:(master) git glog -1
* f4f6187  (HEAD -> master, origin/master, origin/HEAD) Vagrant support to test cp across mountpoints. (2 days ago) <Dexter Plameras>
➜  kick git:(master) git 
➜  kick git:(master) go install -v ./cmd/kick             
# github.com/mattn/go-sqlite3
sqlite3-binding.c: In function ‘sqlite3SelectNew’:
sqlite3-binding.c:128049:10: warning: function may return address of local variable [-Wreturn-local-addr]
128049 |   return pNew;
       |          ^~~~
sqlite3-binding.c:128009:10: note: declared here
128009 |   Select standin;
       |          ^~~~~~~
➜  kick git:(master) kick -v
1.1.0
➜  kick git:(master) kick setup
2021/11/09 09:41:14 can not migrate database: constraints not implemented on sqlite, consider using DisableForeignKeyConstraintWhenMigrating, more details https://github.com/go-gorm/gorm/wiki/GORM-V2-Release-Note-Draft#all-new-migrator
➜  kick git:(master) kick install go git@github.com:kick-project/template-go.git
➜  kick git:(master) kick setup                                                 
2021/11/09 09:41:22 can not migrate database: constraints not implemented on sqlite, consider using DisableForeignKeyConstraintWhenMigrating, more details https://github.com/go-gorm/gorm/wiki/GORM-V2-Release-Note-Draft#all-new-migrator
➜  kick git:(master) kick init
Usage:
    kick init repo <name> [<path>]
    kick init template <name> [<path>]
➜  kick git:(master) kick start go ~/projects/go/kicked-go

2021/11/09 09:41:38 /home/daniel/projects/external/kick/internal/resources/sync/sync.go:194 table installed has no column named created_at
[0.685ms] [rows:0] INSERT OR REPLACE INTO `installed` (`created_at`,`updated_at`,`deleted_at`,`handle`,`template`,`origin`,`url`,`vcs_ref`,`desc`,`time`) VALUES ("2021-11-09 09:41:38.895","2021-11-09 09:41:38.895",NULL,"go","","","git@github.com:kick-project/template-go.git","","Direct installation","2021-11-09 09:41:37.796")
2021/11/09 09:41:38 table installed has no column named created_at
panic: table installed has no column named created_at

goroutine 1 [running]:
github.com/kick-project/kick/internal/resources/errs.Panic({0xbe6840, 0xc00009c930})
	/home/daniel/projects/external/kick/internal/resources/errs/errs.go:106 +0x1c5
github.com/kick-project/kick/internal/resources/sync.(*Sync).Files(0xc000398240)
	/home/daniel/projects/external/kick/internal/resources/sync/sync.go:195 +0x610
github.com/kick-project/kick/internal/services/start.Start.Start({0xc00037ac80, 0xc000360e10, 0xc0003418c0, {0xbe5940, 0xc000368088}, {0xbe6200, 0xc000010020}, {0xbe6200, 0xc000010018}, {0xbee6a0, ...}, ...}, ...)
	/home/daniel/projects/external/kick/internal/services/start/start.go:68 +0xde
github.com/kick-project/kick/internal/subcmds/startcmd.Start({0xc0000200d0, 0x3, 0x3}, 0x1)
	/home/daniel/projects/external/kick/internal/subcmds/startcmd/startcmd.go:46 +0x187
main.main()
	/home/daniel/projects/external/kick/cmd/kick/main.go:53 +0x1e5
➜  kick git:(master) uname -a
Linux e590 5.10.70-1-MANJARO #1 SMP PREEMPT Thu Sep 30 15:29:01 UTC 2021 x86_64 GNU/Linux
➜  kick git:(master) mount
proc on /proc type proc (rw,nosuid,nodev,noexec,relatime)
sys on /sys type sysfs (rw,nosuid,nodev,noexec,relatime)
dev on /dev type devtmpfs (rw,nosuid,relatime,size=16282412k,nr_inodes=4070603,mode=755,inode64)
run on /run type tmpfs (rw,nosuid,nodev,relatime,mode=755,inode64)
efivarfs on /sys/firmware/efi/efivars type efivarfs (rw,nosuid,nodev,noexec,relatime)
/dev/mapper/luks-4f8c0fd3-4a07-4545-b045-a37a0375f9e2 on / type ext4 (rw,noatime)
securityfs on /sys/kernel/security type securityfs (rw,nosuid,nodev,noexec,relatime)
tmpfs on /dev/shm type tmpfs (rw,nosuid,nodev,inode64)
devpts on /dev/pts type devpts (rw,nosuid,noexec,relatime,gid=5,mode=620,ptmxmode=000)
cgroup2 on /sys/fs/cgroup type cgroup2 (rw,nosuid,nodev,noexec,relatime,nsdelegate,memory_recursiveprot)
pstore on /sys/fs/pstore type pstore (rw,nosuid,nodev,noexec,relatime)
none on /sys/fs/bpf type bpf (rw,nosuid,nodev,noexec,relatime,mode=700)
systemd-1 on /proc/sys/fs/binfmt_misc type autofs (rw,relatime,fd=30,pgrp=1,timeout=0,minproto=5,maxproto=5,direct,pipe_ino=16644)
debugfs on /sys/kernel/debug type debugfs (rw,nosuid,nodev,noexec,relatime)
hugetlbfs on /dev/hugepages type hugetlbfs (rw,relatime,pagesize=2M)
mqueue on /dev/mqueue type mqueue (rw,nosuid,nodev,noexec,relatime)
tracefs on /sys/kernel/tracing type tracefs (rw,nosuid,nodev,noexec,relatime)
tmpfs on /tmp type tmpfs (rw,nosuid,nodev,size=16294564k,nr_inodes=409600,inode64)
binfmt_misc on /proc/sys/fs/binfmt_misc type binfmt_misc (rw,nosuid,nodev,noexec,relatime)
configfs on /sys/kernel/config type configfs (rw,nosuid,nodev,noexec,relatime)
fusectl on /sys/fs/fuse/connections type fusectl (rw,nosuid,nodev,noexec,relatime)
/var/lib/snapd/snaps/bare_5.snap on /var/lib/snapd/snap/bare/5 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/core_10126.snap on /var/lib/snapd/snap/core/10126 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/screen_75.snap on /var/lib/snapd/snap/screen/75 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/gtk-common-themes_1519.snap on /var/lib/snapd/snap/gtk-common-themes/1519 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/core_11993.snap on /var/lib/snapd/snap/core/11993 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/core18_2246.snap on /var/lib/snapd/snap/core18/2246 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/kreya_68.snap on /var/lib/snapd/snap/kreya/68 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/hello-world_29.snap on /var/lib/snapd/snap/hello-world/29 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/gnome-3-34-1804_72.snap on /var/lib/snapd/snap/gnome-3-34-1804/72 type squashfs (ro,nodev,relatime,x-gdu.hide)
/dev/nvme0n1p1 on /boot/efi type vfat (rw,noatime,fmask=0022,dmask=0022,codepage=437,iocharset=ascii,shortname=mixed,utf8,errors=remount-ro)
tmpfs on /run/user/1000 type tmpfs (rw,nosuid,nodev,relatime,size=3258912k,nr_inodes=814728,mode=700,uid=1000,gid=1000,inode64)
gvfsd-fuse on /run/user/1000/gvfs type fuse.gvfsd-fuse (rw,nosuid,nodev,relatime,user_id=1000,group_id=1000)
jetbrains-toolbox on /tmp/.mount_jetbraEI5L3h type fuse.jetbrains-toolbox (ro,nosuid,nodev,relatime,user_id=1000,group_id=1000)
portal on /run/user/1000/doc type fuse.portal (rw,nosuid,nodev,relatime,user_id=1000,group_id=1000)
run on /run/snapd/ns type tmpfs (rw,nosuid,nodev,relatime,mode=755,inode64)
nsfs on /run/snapd/ns/kreya.mnt type nsfs (rw)
➜  kick git:(master) pamac show sqlite                       
Available actions:
  pamac --version     
  pamac --help, -h     [action]
  pamac search         [options] <package(s)>
  pamac list           [options] <package(s)>
  pamac info           [options] <package(s)>
  pamac install        [options] <package(s)>
  pamac reinstall      [options] <package(s)>
  pamac remove         [options] [package(s)]
  pamac checkupdates   [options]
  pamac update,upgrade [options]
  pamac clone          [options] <package(s)>
  pamac build          [options] [package(s)]
  pamac clean          [options]
➜  kick git:(master) pamac info sqlite
Name                  : sqlite
Version               : 3.36.0-1
Description           : A C library that implements an SQL database engine
URL                   : https://www.sqlite.org/
Licenses              : custom:Public Domain
Repository            : core
Installed Size        : 10,0 MB
Depends On            : readline zlib
Make Dependencies     : tcl readline zlib
Required By           : colord docker filezilla gnupg gom ipython lib32-sqlite libaccounts-glib libchamplain libsoup libsoup3 nss python-apsw python2 qt5-base tracker tracker3 webkit2gtk
Optional For          : apr-util nfs-utils python
Provides              : sqlite3=3.36.0
Replaces              : sqlite3
Packager              : Andreas Radke <andyrtr@archlinux.org>
Build Date            : 19.06.2021
Install Date          : 27.07.2021
Install Reason        : Installed as a dependency for another package
Signatures            : Yes

➜  kick git:(master)
```

However after removing ~/.kick directory I am able to start a project but it seems to be almost empty:
```shell
➜  kick git:(master) rm -rf ~/.kick
➜  kick git:(master) make depdev
/bin/bash: line 1: maker: command not found
make: *** [Makefile:343: depdev] Error 127
➜  kick git:(master) 
➜  kick git:(master) kick install go git@github.com:kick-project/template-go.git
not initialized. please run "kick setup" to initialize configuration
➜  kick git:(master) kick setup                                                 
➜  kick git:(master) kick install go git@github.com:kick-project/template-go.git
installed handle:go -> location:git@github.com:kick-project/template-go.git
➜  kick git:(master) kick start go ~/projects/go/kicked-go                      
created project handle:go -> project:/home/daniel/projects/go/kicked-go
➜  kick git:(master) l ~/projects/
➜  kick git:(master) l ~/projects/go/kicked-go 
total 20K
drwxr-xr-x 3 daniel daniel 4,0K 11-09 09:45 .
drwxr-xr-x 7 daniel daniel 4,0K 11-09 09:45 ..
-rw-r--r-- 1 daniel daniel  110 11-09 09:45 .editorconfig
-rw-r--r-- 1 daniel daniel  168 11-09 09:45 .env
drwxr-xr-x 3 daniel daniel 4,0K 11-09 09:45 .github
➜  kick git:(master) l /tmp/kick-go*         
/tmp/kick-go-291155904:
total 52K
drwx------  5 daniel daniel  340 11-09 09:19 .
drwxrwxrwt 32 root   root   1,1K 11-09 09:45 ..
drwxr-xr-x  3 daniel daniel   60 11-09 09:19 cmd
-rw-r--r--  1 daniel daniel  110 11-09 09:19 .editorconfig
-rw-------  1 daniel daniel  168 11-09 09:19 .env
drwxr-xr-x  3 daniel daniel   60 11-09 09:19 .github
-rw-------  1 daniel daniel  305 11-09 09:19 .gitignore
-rw-r--r--  1 daniel daniel  666 11-09 09:19 goinstalls.txt
-rw-------  1 daniel daniel   24 11-09 09:19 go.mod
drwxr-xr-x  3 daniel daniel  100 11-09 09:19 internal
-rw-r--r--  1 daniel daniel  390 11-09 09:19 kicked-go.rb.in
-rw-r--r--  1 daniel daniel   57 11-09 09:19 .kick.yml
-rw-r--r--  1 daniel daniel 7,7K 11-09 09:19 Makefile
-rw-------  1 daniel daniel   10 11-09 09:19 NAME
-rw-r--r--  1 daniel daniel  556 11-09 09:19 nfpm.yaml.in
-rw-------  1 daniel daniel   12 11-09 09:19 README.md
-rw-r--r--  1 daniel daniel    6 11-09 09:19 VERSION

/tmp/kick-go-3862622893:
total 40K
drwx------  5 daniel daniel  280 11-09 09:45 .
drwxrwxrwt 32 root   root   1,1K 11-09 09:45 ..
drwxr-xr-x  3 daniel daniel   60 11-09 09:45 cmd
drwxr-xr-x  2 daniel daniel   40 11-09 09:45 .github
-rw-------  1 daniel daniel  305 11-09 09:45 .gitignore
-rw-r--r--  1 daniel daniel  666 11-09 09:45 goinstalls.txt
-rw-------  1 daniel daniel   24 11-09 09:45 go.mod
drwxr-xr-x  3 daniel daniel  100 11-09 09:45 internal
-rw-r--r--  1 daniel daniel  390 11-09 09:45 kicked-go.rb.in
-rw-r--r--  1 daniel daniel 7,7K 11-09 09:45 Makefile
-rw-------  1 daniel daniel   10 11-09 09:45 NAME
-rw-r--r--  1 daniel daniel  556 11-09 09:45 nfpm.yaml.in
-rw-------  1 daniel daniel   12 11-09 09:45 README.md
-rw-r--r--  1 daniel daniel    6 11-09 09:45 VERSION
➜  kick git:(master)
```

However starting project under the /tmp mount point works:
```shell
➜  kicked-go kick start go /tmp/kicked-go         
created project handle:go -> project:/tmp/kicked-go
➜  kicked-go l /tmp/kicked-go
total 48K
drwx------  5 daniel daniel  320 11-09 09:50 .
drwxrwxrwt 33 root   root   1,2K 11-09 09:50 ..
drwxr-xr-x  3 daniel daniel   60 11-09 09:50 cmd
-rw-r--r--  1 daniel daniel  110 11-09 09:50 .editorconfig
-rw-------  1 daniel daniel  168 11-09 09:50 .env
drwxr-xr-x  3 daniel daniel   60 11-09 09:50 .github
-rw-------  1 daniel daniel  305 11-09 09:50 .gitignore
-rw-r--r--  1 daniel daniel  666 11-09 09:50 goinstalls.txt
-rw-------  1 daniel daniel   24 11-09 09:50 go.mod
drwxr-xr-x  3 daniel daniel  100 11-09 09:50 internal
-rw-r--r--  1 daniel daniel  390 11-09 09:50 kicked-go.rb.in
-rw-r--r--  1 daniel daniel 7,7K 11-09 09:50 Makefile
-rw-------  1 daniel daniel   10 11-09 09:50 NAME
-rw-r--r--  1 daniel daniel  556 11-09 09:50 nfpm.yaml.in
-rw-------  1 daniel daniel   12 11-09 09:50 README.md
-rw-r--r--  1 daniel daniel    6 11-09 09:50 VERSION
```

To sum up:

1. Migrations do not work for me probably due to too new sqlite version
2. Starting project in different mount point than /tmp creates only subset of files
3. Starting project in the same mount point as /tmp works well. 
4. Makefile doesn't work for me, because I don't have `maker` command (which I cannot find in pamac / AUR)
5. I use Manjaro BTW

**Comment by dexterp** [comment]
Hello @Fenthick ,

I've made some changes that I think will fix the issues you have outlined. 
> 1. Migrations do not work for me probably due to too new sqlite version

Since you removed the `~/.kick` directory and the subsequent sqlite3 database file. `~/.kick/metadata/metadata.db` this will avoid the need for migration.
BTW you can backup the `~/.kick/templates.yml` and `~/.kick/config.yml` files and copy them to any `~/.kick/ ` installation. You can remove the other files they will get recreated. The `metadata.db` file is just easier to programatically query then a yaml file. 

> 2. Starting project in different mount point than /tmp creates only subset of files

This should be fixed. To build see the instructions [BUILD.md](https://github.com/kick-project/kick/blob/master/BUILD.md)
I've tested this in [Vagrant](https://www.vagrantup.com/).

> 4. Makefile doesn't work for me, because I don't have maker command (which I cannot find in pamac / AUR)

This is installed using `make depsdev`. These are Go CLIs (gocyclo, maker, golint ...) that are installed in your `$GOPATH/bin`. See [BUILD.md](https://github.com/kick-project/kick/blob/master/BUILD.md) for the complete picture.

Let me know how it goes. If it works for you, I will release `1.1.0`.

**Comment by danielfurman** [comment]
Installed latest `f731681` version and it works now:

```zsh
➜  external kick install go https://github.com/kick-project/template-go.git
installed handle:go -> location:https://github.com/kick-project/template-go.git
➜  external kick start go ~/projects/external/kicked-go                    
created project handle:go -> project:/home/daniel/projects/external/kicked-go
➜  external l kicked-go 
total 68K
drwxr-xr-x  5 daniel daniel 4,0K 12-09 12:39 .
drwxrwxr-x 29 daniel daniel 4,0K 12-09 12:39 ..
drwxr-xr-x  3 daniel daniel 4,0K 12-09 12:39 cmd
-rw-r--r--  1 daniel daniel  110 12-09 12:39 .editorconfig
-rw-r--r--  1 daniel daniel  168 12-09 12:39 .env
drwxr-xr-x  3 daniel daniel 4,0K 12-09 12:39 .github
-rw-r--r--  1 daniel daniel  305 12-09 12:39 .gitignore
-rw-r--r--  1 daniel daniel  666 12-09 12:39 goinstalls.txt
-rw-r--r--  1 daniel daniel   24 12-09 12:39 go.mod
drwxr-xr-x  3 daniel daniel 4,0K 12-09 12:39 internal
-rw-r--r--  1 daniel daniel  390 12-09 12:39 kicked-go.rb.in
-rw-r--r--  1 daniel daniel 7,7K 12-09 12:39 Makefile
-rw-r--r--  1 daniel daniel   10 12-09 12:39 NAME
-rw-r--r--  1 daniel daniel  556 12-09 12:39 nfpm.yaml.in
-rw-r--r--  1 daniel daniel   12 12-09 12:39 README.md
-rw-r--r--  1 daniel daniel    6 12-09 12:39 VERSION
```
The issue can be closed.

**Comment by dexterp** [comment]
Fixed in release 1.1.0

Thanks for your attention.

I'm aiming to get enough stars and watches to add this project to homebrew.
If you have any colleagues that might find this tool useful, please consider sending them a link and asking them to star or watch the project.

Kind Regards
Dexter
